mbq/atomiq/producers.py

```diff
@@ -83,6 +92,15 @@ class SQSProducer(BaseProducer):
     required_dependencies = ['boto3']
 
     def _create_task(self, queue_url, payload):
+        if not queue_url or not isinstance(queue_url, str):
+            raise ValueError(
+                'Atomiq sqs_publish got a non string "queue_url" argument: {}'.format(queue_url)
```

Comment:
Applies to all of the errors: 

1. why use the "public" name here even if it's not defined here?
2. it's clearer to say `x must be y` not, `x is non-y` 
3. the type check should be enough, don't need to check for truthiness too.

Example of what I think think would be good.

```python
if not isinstance(queue_url, str):
    raise ValueError('`queue_url` must be a str (got {})'.format(type(queue_url)))
```

Reply:
> don't need to check for truthiness too.
Basically I want to check that (for example) we're not passing in empty string for `topic_arn`. Don't you think is a good thing to check for?

I agree with your other points 👍